client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -441,13 +458,7 @@ class EmailProvidersComparison extends React.Component {
 					comment:
 						'%(titanProductName) is the name of the product, which should be "Professional Email" translated',
 			  } )
-			: translate( 'Add %(titanProductName)s', {
-					args: {
-						titanProductName: getTitanProductName(),
-					},
-					comment:
-						'%(titanProductName) is the name of the product, which should be "Professional Email" translated',
-			  } );
+			: translate( 'Add Email' );
```

Comment:
Why the change here? This will affect all uses of this component, which feels like it's out of scope for now.

Reply:
The Figma file had this copy for the flow but I made an incorrect assumption that this will be the case for other screens. Should I just revert this back to what it was before (i.e _Add Professional Email_) or should I add a prop for the parent to customize this text?